backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/RunVmCommandBase.java

```diff
@@ -298,14 +298,17 @@ protected boolean connectLunDisks(Guid hostId) {
                         .distinct()
                         .collect(groupingBy(StorageServerConnections::getStorageType, toList()));
 
+        List<String> deviceList = lunDisks.stream().map(d -> d.getLun().getLUNId()).collect(Collectors.toList());
```

Comment:
how about lunIds instead of deviceList?

Reply:
You mean for this variable specifically, or for the API parameter name?

The variable is named to match the API parameter. I am not against the change, is more specific and clear, but if so I'll change it in all the occurrences, including also in vdsm side.